Abstract Syntax and Semantics Document/index.html

```diff
@@ -151,9 +151,21 @@
     
     <p>Let T denote the set of temporal entities. This specification is neutral regarding the formal specification of temporal entities.</p>
     
-    <h4>Instants and Intervals</h4>
+    <h4>Instants and Intervals</h4> 
+
+     An instant is an instantaneous point of time on underlying axis and is isomorphic to numbers. 
+     Henceforth a time interval is the interval between two instants. An object is either a further RDF triple or a set of triples. As a consequence a temporal entity is an entity associated with a timestamp or a time interval[Dyreson94,Allen83]. <br>
+     <p class="ednote">Add a out of scope section</p> 
+     <p class="ednote">Add Stream Profiles and Hierarchies</p>
+     
+    <aside class="example" title="OWL-Time">
+    
+      <p>An ontology that follows these specifications is the OWLTime ontology (see <a href="http://www.w3.org/TR/owl-time/"></a>). It defines all necessary temporal RDF concepts, such as instants, intervals and temporalEntities, and relations thereon. 
+      </p>
```

Comment:
"follows these specifications"
could be replaced by
"contains both the concept of instants and time intervals"
because we are not making a requirement that these concepts be included in an acceptable temporal ontology.